### DISCUSSION TITLE
properties pannel has move to preview only ?

### DISCUSSION THREAD

**Discussion by moolight-seashell** [slint-ui/slint#5778] [Q&A] [answered]
on vscode my propertie pannel is gone and am not sure why
![image](https://github.com/user-attachments/assets/b8c43c2a-8919-44c9-a7b2-a8f9c52401d1)
the pannel is not looking hiden by vscode

**Comment by tronical** [comment, marked as answer]
Yes, instead of two property panels there’s only one in the preview now.
